targets/simple_switch/primitives.cpp

```diff
@@ -59,8 +59,18 @@ class modify_field_rng_uniform
     using hash = std::hash<std::thread::id>;
     static thread_local engine generator(hash()(std::this_thread::get_id()));
     using distrib64 = std::uniform_int_distribution<uint64_t>;
-    distrib64 distribution(b.get_uint64(), e.get_uint64());
-    f.set(distribution(generator));
+    uint64_t lo = b.get_uint64();
+    uint64_t hi = e.get_uint64();
+    std::string warning = "";
+    if (lo > hi) {
+        warning = " warning: result is not specified when lo > hi";
```

Comment:
Mainly because of my ignorance of the code.  I can make updates based on this and your other comments.  I wasn't actually expecting these changes to pass review, but didn't know a faster way than giving something undesirable, to find out what was better.